src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -47,30 +71,64 @@ export class EventManager implements IEventManager {
     this.#subscriptionManager = new SubscriptionManager();
   }
 
+  /**
+   * Returns consistent key to be used to access values maps.
+   * @param eventName
+   * @param browsingContext
+   * @param channel
+   * @private
+   */
+  static #getMapKey(
+    eventName: string,
+    browsingContext: CommonDataTypes.BrowsingContext | null,
+    channel: string | null | undefined = undefined
+  ) {
+    return JSON.stringify({ eventName, browsingContext, channel });
+  }
+
   async sendEvent(
     event: Message.EventMessage,
     contextId: CommonDataTypes.BrowsingContext | null
   ): Promise<void> {
+    const eventWrapper = new IdWrapper<Message.EventMessage>(event);
     const sortedChannels =
       this.#subscriptionManager.getChannelsSubscribedToEvent(
         event.method,
         contextId
       );
-
+    // Buffers event if needed.
```

Comment:
This comment seems redundant to me, I'd remove it.

Reply:
agree )